protocol/src/main/java/org/fido/iot/protocol/epid/EpidHttpClient.java

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Intel Corporation
+// SPDX-License-Identifier: Apache 2.0
+
+package org.fido.iot.protocol.epid;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.time.Duration;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class EpidHttpClient {
+
+  private static ExecutorService executor = Executors.newCachedThreadPool(r -> {
+    Thread t = Executors.defaultThreadFactory().newThread(r);
+    t.setDaemon(true);
+    return t;
+  });
+  private static final long httpRequestTimeout = Duration.ofSeconds(10).getSeconds();
```

Comment:
Why not a Duration?

Reply:
We'd need to convert it to something primitive like sec/ms etc later anyways at a couple of places. So converted this once here.